src/ui/atoms/SelectField.tsx

```diff
@@ -53,6 +54,7 @@ const SelectField = ({
                 onChange={onChange}
                 isMulti={multi}
                 defaultValue={defaultValue}
+                isSearchable={searchable}
```

Comment:
does forwarding this prop need unit tests?

Reply:
As part of https://github.com/libero/reviewer-client/issues/97 we should probably make this component just extend the prop types of whatever off the shelf select we're using (if we don't make our own) and forward those props to the component.

Reply:
Testing this looks to be a bit of a pain as it'll likely slip into testing implementation detail if we aren't careful. Maybe we can leave without the added test (as we're just forwarding the prop) on the understanding we'll be re-writing select very soon.